ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -150,13 +152,14 @@ public void handleGet(RestRequest restRequest, RestResponseChannel restResponseC
           case UserMetadata:
             requestMetrics = adminMetrics.getUserMetadataMetrics;
             break;
+          case Replicas:
```

Comment:
I don't think a default case is required since there are only three sub resources. Another way of thinking about this - what do I do in the default case?